modules/nf-core/glimpse2/chunk/tests/main.nf.test

```diff
@@ -55,10 +53,40 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out).match() }
+                { assert snapshot(
+                    process.out.chunk_chr,
+                    file(process.out.chunk_chr[0][1]).name, // Result not stable when using a map
```

Comment:
when using a map? As in, the output file depends on the order of the input samples?

Reply:
Sorry my bad, I had this issue previously due to a wrong region initialization.
I've updated the code.